bridge/client/app/_components/ktb-time-input/ktb-time-input.component.html

```diff
@@ -0,0 +1,56 @@
+<dt-label *ngIf="label" class="dt-form-field-label" [ngClass]="{ required: required }">Timeframe</dt-label>
+<div
+  class="timeframe-input"
+  [ngClass]="{ focused: isFocused }"
+  (click)="focusClick($event, hourInput)"
+  (focus)="hourInput.focus()"
+>
+  <form [formGroup]="timeForm" uitestid="keptn-time-input">
+    <input
+      #hourInput
+      dtInput
+      placeholder="hh"
+      type="number"
+      (focusin)="isFocused = true"
+      (focusout)="isFocused = false"
+      formControlName="hours"
+      (input)="validateInput('hours', 0, 24)"
+      uitestid="keptn-time-input-hours"
+    />:<input
+      dtInput
+      placeholder="mm"
+      (focusin)="isFocused = true"
+      (focusout)="isFocused = false"
+      formControlName="minutes"
+      (input)="validateInput('minutes', 0, 60)"
+      uitestid="keptn-time-input-minutes"
+    /><ng-container *ngIf="secondsEnabled"
+      >:<input
+        dtInput
+        placeholder="ss"
+        (focusin)="isFocused = true"
+        (focusout)="isFocused = false"
+        formControlName="seconds"
+        (input)="validateInput('seconds', 0, 60)"
```

Comment:
Should be 0 to 59.
```suggestion
        (input)="validateInput('seconds', 0, 59)"
```

Reply:
Yes and no for the timeframe there are no limits, but for the datetime picker there are. As discussed I will implement a min / max configuraiton for the time-input.